scripts/import/import_variant_synonyms

```diff
@@ -103,7 +103,8 @@ elsif($source_name eq 'dbSNP') {
       '-driver'  =>  'mysql' 
     ); 
 
-  my $synonyms_output = query_synonyms_data($prod_dbc, 'rs_name,synonym_name,source_id', 'human_synonym'); 
+  my $sql_query = "SELECT rs_name,synonym_name,source_id FROM human_synonym"; 
+  my $synonyms_output = query_synonyms_data($prod_dbc, $sql_query); 
```

Comment:
The data structure made from $synonyms_output assumes synonym_name is unique, but it is not.